examples/2d/elixir_euler_unstructured_quad_wall_bc.jl

```diff
@@ -0,0 +1,86 @@
+
+using Downloads: download
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the compressible Euler equations
+
+equations = CompressibleEulerEquations2D(1.4)
+
+function uniform_flow_state(x, t, equations)
+
+  # set the freestream flow parameters
+  rho_freestream = 1.0
+  u_freestream = 0.3
+  p_freestream = inv(equations.gamma)
+
+  theta = pi / 90.0 # analogous with a two degree angle of attack
+  v1 = u_freestream * cos(theta)
+  v2 = u_freestream * sin(theta)
+
+  return SVector(rho_freestream,
+                 rho_freestream * v1,
+                 rho_freestream * v2,
+                 p_freestream * inv(equations.gamma - 1) + 0.5 * rho_freestream * (v1^2 + v2^2))
```

Comment:
Maybe use `prim2cons` to make this a little bit more clear?

Reply:
Whoops, I forgot that function is exported. I will update this accordingly